### DISCUSSION TITLE
How to contribute/customize the CLIENT

### DISCUSSION THREAD

**Discussion by loliveirawebdev** [lowdefy/lowdefy#1336] [Questions and Help] [answered]
Hello guys
I'm trying to made some tests in the source code, but 'im having some trouble to start the dev environment.
The docs seems that is outdated.

I'm trying to run yarn app:server-dev but it looks like changes in "client" folder are not in charge

**Comment by Gervwyk** [comment, marked as answer]
Hi @lalvesoliveira 

Run yarn then create an `app` folder at the root of the monorepo and inside a lowdefy.yaml with some config and set the lowdefy version to `local`. Then run `yarn app:cli:dev`. Which should start the dev server with the local changes and app config. Unfortunately for now you need to stop and start again if you are making changes to the packages to rebuild the packages.

Thanks for considering to contribute! What changes did you have in mind?

**Reply by loliveirawebdev** [reply]
thank you!
i was just checking some possibilities (use some components created in react native web, for example)...
moreover, my company is planning to adopt the usage of lowdefy for some projects, so this information will be very useful to us to contribute with this project
